.github/workflows/main.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
       # XXX: are these actually unit tests?
       - name: Run Fork Mode Tests
         run: pytest -n auto tests/integration/fork/
+        if: secrets.ALCHEMY_MAINNET_ENDPOINT
```

Comment:
why conditional execution?

Reply:
Cause the environment variable is empty on my prs for some reason, so it's always red